lib/ssr/node_js.ex

```diff
@@ -0,0 +1,17 @@
+defmodule LiveSvelte.SSR.NodeJS do
+  @moduledoc false
+  @behaviour LiveSvelte.SSR
+
+  def render(name, props, slots) do
+    try do
+      NodeJS.call!({"server", "render"}, [name, props, slots])
+    catch
+      :exit, {:noproc, _} -> raise LiveSvelte.SSR.NodeNotConfigured
```

Comment:
I didn't want to change too much, but it might make sense to have a more general-purpose `SSRNotConfigured` exception.

Reply:
Feel free to add this, makes sense!